src/osemgrep/reporting/Matches_report.ml

```diff
@@ -18,6 +18,7 @@ let findings_indent_depth = String.make 12 ' '
 let text_width =
   let max_text_width = 120 in
   let w = Option.value ~default:max_text_width (Terminal_size.get_columns ()) in
+  (* TODO: what is this w - (w - 100) ? What if w <= 5? *)
   if w <= 110 then w - 5 else w - (w - 100)
```

Comment:
taken from cli/src/semgrep/formatter/text.py:

```python
if width <= 110:
    width = width - 5
else:
    width = width - (width - 100)
```

Reply:
I know it's taken from text.py, I still don't really understand why we do that